packages/utilities/exposure-scanning/scan/index.js

```diff
@@ -0,0 +1,77 @@
+const replacementStr = '*';
+
+export const scanText = (text, regex) => {
+  let foundTextMatch = false;
+  const scannedText = text.replace(regex, replacementStr);
+
+  if (scannedText !== text) {
+    foundTextMatch = true;
+  }
```

Comment:
I think you can avoid a mutable variable here

```suggestion
  const scannedText = text.replace(regex, replacementStr);
  const foundTextMatch = scannedText !== text;
```

Reply:
Really good spot